tests/test_feature_aggregate.py

```diff
@@ -400,7 +401,7 @@ def test_aggregate_region_with_weights_inconsistent_index(
 def test_aggregate_region_with_weights_raises(simple_df):
     # carbon price shouldn't be summed but be weighted by emissions
     v = "Price|Carbon"
-    w = "Emissions|CO2"
+    # w = "Emissions|CO2"
```

Comment:
It seems that this can actually be removed, if it was commented out and the test did not break.
```suggestion
```

Reply:
I just wasn't sure if this was kept in for some potential new test in the future, but sure, I can delete it.

Reply:
I suggest leave but uncomment  and use (see other suggestion) for consistency 
```suggestion
    w = "Emissions|CO2"
```